impl/src/main/java/org/jboss/fuse/wsdl2rest/impl/Wsdl2Rest.java

```diff
@@ -66,9 +75,17 @@ public void setJavaOut(Path javaOut) {
         JavaTypeGenerator typeGen = new JavaTypeGenerator(effectiveJavaOut(), wsdlUrl);
         JavaModel javaModel = typeGen.execute();
         
+        if (blueprintContext != null) {
+            Path contextPath = effectiveCamelContext(blueprintContext, Paths.get("wsdl2rest-blueprint-context.xml"));
+            CamelContextGenerator camelGen = new SpringContextGenerator(contextPath);
```

Comment:
why the SpringContextGenerator is used here? I would have expected to have the BlueprintContextGenerator.

Reply:
Of course, thanks ;-)